src/components/ResourceList/components/FilterControl/tests/FilterControl.test.tsx

```diff
@@ -637,6 +637,37 @@ describe('<FilterControl />', () => {
     });
   });
 
+  describe('placeholder', () => {
+    it('renders default text if no placeholder passed.', () => {
+      const filterControl = mountWithAppProvider(
+        <Provider value={mockDefaultContext}>
+          <FilterControl {...mockDefaultProps} filters={mockFilters} />
+        </Provider>,
+      );
+
+      expect(filterControl.find(TextField).prop('placeholder')).toBe(
+        'Search items',
+      );
+    });
+
+    it('renders the placeholder prop value if provided', () => {
+      const placeholder = 'by name, email or phone';
```

Comment:
See this does makes me think it's better to be 100% up to the user what they pass in. Search... was opinionated base we knew what we would render, but having to provide a placeholder that will be prepended by Search requires the consumer to know this.

Reply:
OK, sounds good! So just to confirm I will remove the `Search` part and the `placeholder` can be anything the user wants it to be?

Reply:
Yeah I would think so.